Library/Homebrew/cmd/audit.rb

```diff
@@ -878,12 +878,12 @@ def audit_line(line, lineno)
         problem "`#{$1}` is now unnecessary"
       end
 
-      if line =~ /#\{share\}\/#{formula.name}/
+      if line =~ %r{#\{share\}/#{formula.name}[/'"]}
         problem "Use \#{pkgshare} instead of \#{share}/#{formula.name}"
       end
 
-      if line =~ /share\/"#{formula.name}"/
-        problem "Use pkgshare instead of (share/\"#{formula.name}\")"
```

Comment:
I'm almost certain we'll see some cases of `(pkgshare)` given some of the things we've seen changed oddly in relation to audit suggestions :smile_cat:.

That said I have no strong objection to the change; I'm sure most contributors will understand the intention.